queries/buildings.jinja2

```diff
@@ -1,12 +1,9 @@
 SELECT
     name,
     way_area::bigint AS area,
+    COALESCE("building", "amenity", "shop", "tourism") AS kind,
```

Comment:
Yea, this is happening in a transform at the moment to account for that.

https://github.com/mapzen/TileStache/blob/0e6acb222c32653b2622e732d2745652ee208f52/TileStache/Goodies/VecTiles/transform.py#L156-L164

Maybe what was going on was for these features the values for `amenity`, `shop`, and `tourism` were all null so no kind value was being set.